plugins/sts/src/index.mjs

```diff
@@ -0,0 +1,33 @@
+/**
+ * Plugin maintained by: @architect
+ */
+
+import incomplete from './incomplete.mjs'
+
+const service = 'sts'
+const property = 'STS'
+// const required = true
+const docRoot = 'https://docs.aws.amazon.com/STS/latest/APIReference/'
+
+const GetCallerIdentity = {
+  awsDoc: docRoot + 'API_GetCallerIdentity.html',
+  validate: {},
+  request: () => {
+    return {
+      path: '',
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      payload: 'Action=GetCallerIdentity&Version=2011-06-15',
+    }
+  },
+  response: (payload) => { payload },
```

Comment:
This is incorrect, please see: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/sts/command/GetCallerIdentityCommand/